.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
+<!-- Please search existing Feature request to avoid creating duplicates. -->
+<!-- It is recommended that you start a discussion first in: https://discuss.dgraph.io
+before open a Feature request. That way we keep the Issues track system cleaner and productive. -->
+
+<!-- Fill out below. -->
+
+**Is your feature request related to a problem? Please describe.**
+<!-- Tip: Add a Bug Report reference or a discussion in the https://discuss.dgraph.io -->
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
Let's model this on Go experience reports. They're the best way to figure out what features we should be building.

https://github.com/golang/go/wiki/ExperienceReports

Can you modify the template to reflect questions there?

Reply:
Sure, I'l do.